### PR TITLE
Dz=11

### PR DESCRIPTION
**Pull request by worldwideboi**

### CODE REVIEW

File: dz-11/src/test/java/cc/robotdreams/test/ManAndWomanTests.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import cc.robotdreams.Woman;
+import static org.testng.Assert.assertEquals;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test
+public  class ManAndWomanTests
+{
+    @Test
+    public void testIsRetiredMan() {
```
Comment (yurii-voronenko) [inline dz-11/src/test/java/cc/robotdreams/test/ManAndWomanTests.java:12]: For testing retirement use boundary values, for men it's 64, 65, 66

File: dz-11/src/test/java/cc/robotdreams/test/ManAndWomanTests.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        Man john = new Man("John", "Doe", 70);
+        Assert.assertTrue(john.isRetired());
+
+        john.setAge(50);
+        Assert.assertFalse(john.isRetired());
+
+    }
+
+    @Test
+    public void testIsRetiredWoman() {
```
Comment (yurii-voronenko) [inline dz-11/src/test/java/cc/robotdreams/test/ManAndWomanTests.java:22]: For testing retirement use boundary values, for woman it's 59, 60, 61

File: dz-11/src/test/java/cc/robotdreams/test/ManAndWomanTests.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+            jane.deregisterPartnership(true);
+
+            Assert.assertEquals(john.getPartner(), null);
+            Assert.assertEquals(john.getLastName(), "Doe");
+            Assert.assertEquals(jane.getPartner(), null);
+            Assert.assertEquals(jane.getLastName(), "Smith");
+        }
+
+        public void testSetAge(){
```
Comment (yurii-voronenko) [inline dz-11/src/test/java/cc/robotdreams/test/ManAndWomanTests.java:61]: Annotation @Test absent here

**Review by yurii-voronenko — APPROVED** [review]
LGTM
